draft-ion.adoc

```diff
@@ -52,50 +58,59 @@ However, JSON's simple grammar can support virtually any data model - additional
 context is required for machine clients and servers to interpret data and
 metadata in important ways.
 
-Ion exists to codify specific name/value pairs within a JSON object to better
-facilitate JSON-based data and metadata exchange between machine clients and
-servers. As such, Ion does not define a new data grammar - it is a
-Media Subtype of `application/json`.
+Ion codifies specific name/value pairs within JSON objects to better
+facilitate JSON-based data and metadata exchange between machines.  And while machine interchange is the primary goal, Ion's second most important design goal is to enable intuitive and context-relevant structure so Ion content is very easy to understand for humans, even if they do not know what Ion is.  As such, Ion does not define a new data grammar - it is a Media Subtype of `application/json`.
 
 === Terminology
 
 These terms are defined by this specification:
 
+.Member
+A JSON name/value pair as defined in https://tools.ietf.org/html/rfc7159#section-4[RFC 7159].
+
 .Metadata
-Data about a resource.
+A set of data that provides information about other data.
 
 .Meta Object
-A JSON object that contains resource metadata.
+A JSON object with one or more members that all represent Metadata.
+
+.Root Object
+The single JSON object at the root of an Ion content structure.
 
-.Instance Resource
-A JSON object that represents a single data resource and its metadata.
+.Ion Object
+A JSON object with a Meta Object member named `meta`.
+
+.Resource
+An Ion Object that has one or more members in addition to its `meta` member.
 
 .Collection Resource
```

Comment:
So, this is one of the more abused concepts that is a legacy of Rails et al, right behind `nested resources`, neither of which exist in reality. There are just resources. If you look at the IANA definition of `next` and `prev` it has the idea of a resource that is part of a sequence. @lhazlewood Not sure how you feel about this, but I principally try in this space to not propagate ideas that everyone takes for granted but are not accurate.

I have wondered about how one might address this. For example, by introducing this, you have to tie up both `meta` and `items` in Ion and you limit flexibility. Frankly the resource does not care what aspect the sequence shows up in. You just call next and get it. I understand practically that may be too theoretical. I have wondered if there could be something in the meta associated with a `next` relation that indicates what property it relates to that is sequential and then it is all less hard coded. There are also some aspects of how to use `item` and `collection` relations around that apply to this.

Thoughts?